src/views/Login/Ticket.js

```diff
@@ -143,19 +141,24 @@ export default function Ticket({ className, goHome }) {
           point,
           values.passphrase
         );
-
-        const [isOwner, isTransferProxy] = await Promise.all([
-          azimuth.azimuth.isOwner(
-            _contracts,
-            point,
-            urbitWallet.ownership.keys.address
-          ),
-          azimuth.azimuth.isTransferProxy(
-            _contracts,
-            point,
-            urbitWallet.ownership.keys.address
-          ),
+        const [ownedPoints, incomingPoints] = await Promise.all([
+          getPoints('own', urbitWallet.ownership.keys.address),
+          getPoints('transfer', urbitWallet.ownership.keys.address),
         ]);
+        const isOwner = ownedPoints.length !== 0;
+        const isTransferProxy = incomingPoints.length !== 0;
+        // const [isOwner, isTransferProxy] = await Promise.all([
```

Comment:
@yosoyubik Do we still need this?

Reply:
No, that  can go—I addressed it here https://github.com/urbit/bridge/pull/635/commits/ea51dfbc2dca5e2d5a4098aeb7c8e8cfb5b73956